packages/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.ts

```diff
@@ -701,9 +708,15 @@ export class CurrentActivityPrinter extends ActivityPrinterBase {
     return '[' + color(fullChars + partialChar) + filler + `] (${this.resourcesDone}/${this.resourcesTotal})`;
   }
 
+  private failureReason(activity: StackActivity) {
+    const resourceStatusReason = activity.event.ResourceStatusReason ?? '';
+    const logicalResourceId = activity.event.LogicalResourceId ?? '';
+    return resourceStatusReason.includes('The following hook(s) failed')? resourceStatusReason + ' : ' + this.hookFailureMap.get(logicalResourceId) ?? '' : resourceStatusReason;
+  }
+
   private failureReasonOnNextLine(activity: StackActivity) {
     return hasErrorMessage(activity.event.ResourceStatus ?? '')
-      ? `\n${' '.repeat(TIMESTAMP_WIDTH + STATUS_WIDTH + 6)}${chalk.red(activity.event.ResourceStatusReason ?? '')}`
+      ? `\n${' '.repeat(TIMESTAMP_WIDTH + STATUS_WIDTH + 6)}${chalk.red(this.failureReason(activity) ?? '')}`
```

Comment:
This is only changing one of the two printers. Do we care about this one as well?

https://github.com/aws/aws-cdk/blob/280b876d426522fd043f13ff0b188e29acaa0fbf/packages/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.ts#L551